src/core/client/stream/test/fixtures.ts

```diff
@@ -869,6 +870,61 @@ export const storyWithDeepestReplies = denormalizeStory(
   )
 );
 
+export const replyableComment: GQLComment = {
+  ...comments[0],
+  author: {
+    ...comments[0].author!,
+    username: "replyable comment author",
+  },
+  canReply: true,
+};
+
+export const rejectedComment: GQLComment = {
+  ...comments[1],
+  author: {
+    ...comments[1].author!,
+    username: "rejected comment author",
+  },
+  status: GQLCOMMENT_STATUS.REJECTED,
+};
+
+export const unreplyableComment: GQLComment = {
```

Comment:
Typo: `unreplyable` -> `unrepliable`

I had to check this, but this is indeed the correct spelling, and to boot, it even means what it's supposed to mean!

![image](https://user-images.githubusercontent.com/5751504/183977851-9eee18e9-6cc8-46fa-a95b-cd0420626215.png)

It's also hilarious how it puts a `RARE` marker beside the word cause absolutely nobody is using this word in an every-day sentence haha

P.S. you don't actually have to fix this typo... I just thought it was funny

Reply:
hahaha. im actually down to change this tbh